### PR TITLE
Update: training_models_at_scale.ipynb Notebook

### PR DESCRIPTION
**Pull request by kalyanipande-tudip**
- Added the informative comments with all description in solution notebook.
- Adjusted the instructions for better understanding in solution notebook.
- Related Buganizer: http://b/171861568

### CODE REVIEW

File: courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/training_models_at_scale.ipynb [hunk 1]
```diff
@@ -27,6 +27,7 @@
       "colab": {}
```
Comment (gstripling) [inline courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/training_models_at_scale.ipynb:27]: <p>LIne 1, Python - "p" should be capitalized.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1301/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/training_models_at_scale.ipynb:27]: <p>Capitalize the "p" in Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1301/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/training_models_at_scale.ipynb:27]: Updated the same.

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/training_models_at_scale.ipynb:27]: Updated the same

Comment (gstripling) [reply courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/training_models_at_scale.ipynb:27]: <p>Thanks.</p>

**Pull request merged**